src/v2/Components/v2/ArtworkFilter/ArtworkFilters/TimePeriodFilter.tsx

```diff
@@ -24,6 +24,8 @@ export const TimePeriodFilter: FC<TimePeriodFilterProps> = ({
     periods = allowedPeriods.map(name => ({ name }))
   }
 
+  if (!periods.length) return null
```

Comment:
`periods` is already transformed into an array, since we have an allow-list of periods we filter the aggregations through